examples/train_mnist.py

```diff
@@ -1,29 +1,27 @@
-import numpy as np
-
-from vulcanai.net import Network
-
-import theano.tensor as T
 
+from net import Network
```

Comment:
I think the package structure got messed up somewhere, cause I can't run anything from within the examples folder (or outside) without an import error